src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -58,8 +58,8 @@ public AddCommand parse(String args) throws ParseException {
         }
 
         Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
-        Company company = new Company("Placeholder");
-        JobTitle jobTitle = new JobTitle("Placeholder");
+        Company company = null;
+        JobTitle jobTitle = null;
```

Comment:
Do we need to explicitly set these objects as `null`?

Reply:
If I don't initialize them the code will not compile because "variable 'x' might not have been initialised